.eslintrc.js

```diff
@@ -26,7 +26,6 @@ module.exports = {
     'import/order': 'error',
     'no-duplicate-imports': 'error',
     'no-implicit-globals': 'error',
-    'no-magic-numbers': 'error',
```

Comment:
i could just disable this rule inline for tests, but it was complaining about using numbers like `.toHaveBeenCalledTimes(1)`, which seemed excessive. this rule seems more useful for application code than for a component library, but i'm also fine with re-enabling and disabling the rule inline if people have feelings about this